Cargo.toml

```diff
@@ -60,6 +58,7 @@ hex-literal = "0.4.1"
 lazy_static = "1.4"
 proptest = "1.2.0"
 tokio = { version = "1.33", features = ["macros", "rt-multi-thread"] }
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", features = ["cli"] }
```

Comment:
Build seems to indicate that this change doesn't work. https://github.com/IronCoreLabs/ironcore-alloy/actions/runs/8884446697/job/24393687125?pr=114#step:7:222